EIPS/eip-5089.md

```diff
@@ -0,0 +1,340 @@
+---
+eip: 5089
+title: Principal Token Standard
+description: A standard for principal tokens (zero-coupon tokens) that are redeemable for a single underlying ERC-20 token at a future timestamp.
```

Comment:
We discourage "standard" in the title and description of EIPs:

```suggestion
title: Principal Token
description: Principal tokens (zero-coupon tokens) are redeemable for a single underlying ERC-20 token at a future timestamp.
```

Reply:
Fixed